pkg/projects/dir.props

```diff
@@ -96,6 +96,38 @@
     </BuildRID>
   </ItemGroup>
 
+  <!-- ================== Section for Universal Windows Platform package ================== -->
```

Comment:
This doesn't need to be here.  Just put it in some dir.props in the UWP folder.  Also you could just set both of these before importing.  The PackageRID property group already has a condition to do nothing if set,  you could put the same sort of thing on the OfficialBuildRID itemgroup.